src/app/home/home.component.ts

```diff
@@ -25,6 +32,32 @@ export class HomeComponent {
     this.page = page.pageIndex;
   }
 
+  onTabChange(tab: MatTabChangeEvent): void {
+    const index = tab.index;
+    const tabName = Object.keys(this.catalogueService.CONF.tabs)[index];
+    const path = `/tab${this.catalogueService.CONF.tabs[tabName].path}`;
+    
+    if (this.location.path() !== path) {
+      this.location.go(path);
+    }
+  }
+  
+  get isCatalogueLoading(): boolean {
+    return this.catalogueService.isLoading;
+  }
+
+  get tabIndex(): number {
+    if(this.isCatalogueLoading)
+      return -1;
+
+    const tabName = this.route.snapshot.paramMap.get('tab');
+    return tabName ? this.catalogueService.confTabPaths.indexOf(`/${tabName}`) : 0;
+  }
```

Comment:
not sure if this should be in the home component, I think it is better to move this function in the catalogueService, becauseit can be useful in multiple components